application.py

```diff
@@ -1,15 +1,203 @@
+        results = await asyncio.gather(*tasks)
+    return results
+
+
+def package_fire_history(fire_hist_response):
+    """Package fire history data in dict"""
+    fire_hist_package = {}
+    if fire_hist_response["features"] == []:
+        fire_hist_package["fire history"] = "There is no fire history at this location."
+    else:
+        fire_hist_package["Fire Year"] = fire_hist_response["features"][0][
+            "properties"
+        ]["FIREYEAR"]
+        fire_hist_package["Fire Name"] = fire_hist_response["features"][0][
+            "properties"
+        ]["NAME"]
+    return fire_hist_package
+
+
+def package_flammability(flammability_response):
+    """Package flammability data in dict"""
+    flammability_package = {}
+    if flammability_response["features"] == []:
+        flammability_package[
+            "flammability"
+        ] = "There is no relative flammability projection at this location."
+    else:
+        flammability_package["Relative Flammability Index"] = flammability_response[
+            "features"
+        ][0]["properties"]["GRAY_INDEX"]
+    return flammability_package
+
+
+def package_snow(snow_response):
+    """Package snow cover data in dict"""
+    snow_package = {}
+    if snow_response["features"] == []:
+        snow_package["is_snow"] = "There is no snow information at this location."
+    else:
+        snow_package["is_snow"] = snow_status[
+            snow_response["features"][0]["properties"]["GRAY_INDEX"]
+        ]
+    return snow_package
+
+
+def package_fire_danger(fire_danger_response):
+    """Package fire danger data in dict"""
+    fire_danger_package = {}
+    if fire_danger_response["features"] == []:
+        fire_danger_package[
+            "fire_danger"
+        ] = "There is no fire danger information at this location."
+    else:
+        fire_danger_package["code"] = fire_danger_response["features"][0]["properties"][
+            "GRAY_INDEX"
+        ]
+        fire_danger_package["type"] = smokey_bear_names[fire_danger_package["code"]]
+        fire_danger_package["color"] = smokey_bear_styles[fire_danger_package["code"]]
+    return fire_danger_package
+
+
+def package_landcover(landcover_response):
+    """Package landcover data in dict"""
+    landcover_package = {}
+    if landcover_response["features"] == []:
+        landcover_package[
+            "landcover"
+        ] = "There is no landcover information at this location."
+    else:
+        code = landcover_response["features"][0]["properties"]["PALETTE_INDEX"]
+        landcover_package["code"] = code
+        landcover_package["type"] = landcover_names[code]["type"]
+        landcover_package["color"] = landcover_names[code]["color"]
+    return landcover_package
+
+
+def validate_latlon(lat, lon):
+    """Validate the lat and lon values,
+    return bool for validity"""
+    try:
+        lat_numeric = isinstance(int(float(lat)), int) or isinstance(float(lat), float)
```

Comment:
@charparr I'm a little confused about these lines. Shouldn't these always return true if there is no error in the initial conversion from float() or int()?

Reply:
These two lines are left over from testing the code and should be deleted. The inequalities below call `float()` within the `try`:. That should ensure it is either float, int, or string that is convert-able, and so should be able to be placed in the URL string.

Reply:
commit added to remove these.